app/models/item.rb

```diff
@@ -0,0 +1,14 @@
+class Item < ApplicationRecord
+  belongs_to :category
+  validates_presence_of :name
+  validates_uniqueness_of :name
+
+  before_validation :downcase_name
+
+  def downcase_name
+    if self.name?
```

Comment:
It works! I had tried that earlier, but it appeared to not behave correctly which is why I did the downcase.